tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -1161,48 +1161,46 @@ describe("Service Form Modal", function() {
         });
 
         it('Should add new set of form fields when "Add Local Volume" link clicked', function() {
+          cy.get("@tabView").contains("Add Local Volume").click();
+
           cy
-            .get("@tabView")
-            .contains(".form-group", "Volume Type")
-            .within(function() {
-              cy.get("select").should("have.attr", "name", "volumes.0.type");
-            });
+            .get('.form-group-heading-content[title="Volume Type"]')
+            .should("have.length", 2);
         });
 
         it('Should add new set of form fields when "Persistent Volume" is selected as volume type', function() {
+          cy.get("@tabView").contains(".form-group", "Volume Type");
+          cy.get("@tabView").find(".button.dropdown-toggle").click();
           cy
-            .get("@tabView")
-            .find('select[name="volumes.0.type"]')
-            .select("PERSISTENT");
-
-          // Size input
-          cy
-            .get("@tabView")
-            .find('.form-control[name="volumes.0.size"]')
-            .should("exist");
-
-          // Container Path input
+            .contains(".dropdown-select-item-title", "Local Persistent Volume")
+            .click();
+          cy.get('.form-control[name="localVolumes.0.size"]').should("exist");
           cy
-            .get("@tabView")
-            .find('.form-control[name="volumes.0.containerPath"]')
+            .get('.form-control[name="localVolumes.0.containerPath"]')
             .should("exist");
         });
 
-        it('Should remove "Volume" form fields when remove button clicked', function() {
-          cy
-            .get("@tabView")
-            .find('select[name="volumes.0.type"]')
-            .parents(".panel")
-            .within(function() {
+        it.skip(
```

Comment:
Is this skip here by accident? Otherwise I would love to see a comment explaining why we don't want to run this test right now 👍

Reply:
Wasn't an accident, we had a bug introduced to master where the `x` wasn't dispaying and for that reason I did not like the idea of having the test forced to pass so instead I left this here so people could see it's something we still need to fix.

That being said, the bug was fixed on https://github.com/dcos/dcos-ui/pull/2607